applications/nrf5340_audio/prj.conf

```diff
@@ -39,24 +39,17 @@ CONFIG_NCS_INCLUDE_RPMSG_CHILD_IMAGE=n
 CONFIG_RESET_ON_FATAL_ERROR=n
 
 # Removed configs - maybe needed for CIS
-# CONFIG_BT_PRIVACY=n
-# CONFIG_BT_GATT_CLIENT=y
-# CONFIG_BT_GATT_DYNAMIC_DB=y
-# CONFIG_BT_L2CAP_TX_BUF_COUNT=6
-# CONFIG_BT_RX_STACK_SIZE=2000
-# CONFIG_BT_BUF_ACL_RX_SIZE=259
-# CONFIG_BT_BONDABLE=n
-# CONFIG_BT_AUDIO=y
-# CONFIG_BT_EXT_ADV=y
-# CONFIG_BT_PER_ADV=y
-# CONFIG_BT_ISO_UNICAST=y
-# CONFIG_BT_ISO_BROADCASTER=y
-# CONFIG_BT_ISO_SYNC_RECEIVER=y
-# CONFIG_BT_OBSERVER=y
-# CONFIG_BT_PERIPHERAL=y
-# CONFIG_BT_SMP=y
-# CONFIG_BT_GAP_AUTO_UPDATE_CONN_PARAMS=n
-# CONFIG_BT_AUTO_PHY_UPDATE=n
-# CONFIG_BT_AUTO_DATA_LEN_UPDATE=n
-# CONFIG_BT_ISO_RX_MTU=190
-# CONFIG_BT_ISO_TX_MTU=190
+CONFIG_BT_PRIVACY=n
+CONFIG_BT_GATT_CLIENT=y
+CONFIG_BT_GATT_DYNAMIC_DB=y
+CONFIG_BT_L2CAP_TX_BUF_COUNT=6
+CONFIG_BT_BUF_ACL_RX_SIZE=259
+CONFIG_BT_BONDABLE=n
```

Comment:
Move these to under # Bluetooth

Reply:
Hmm, seems these configuration are only needed for CIS (because of the ACL connection) excepted the CONFIG_BT_ISO_*X_MTU which needed by both CIS and BIS. I remove BT_AUDIO which is duplicated.